chapters/functions/slopes-lines-curves/reading/README.md

```diff
@@ -1 +1,114 @@
 # Slopes of Lines and Curves
+
+## The Slope of a Line
+
+Linear functions produce straight-line graphs.
+In general, a straight line follows the following equation:
+
+$$y = a + bx$$
+
+where $a$ and $b$ are fixed numbers.
+
+The line on the graph is the set of points:
+
+$$\left\\\{ (x,y): x,y \in \mathbb{R}, y = a+bx \right\\\}$$
```

Comment:
```suggestion
$$\left\{ (x,y): x,y \in \mathbb{R}, y = a+bx \right\}$$
```

Reply:
@teodutu , are you sure about this? If I don't also escape the backslash, there is a complaint of rendering on [GitHub](https://github.com/razvand/ccas/blob/functions/slopes-lines-curves/chapters/functions/slopes-lines-curves/reading/read.md).

Reply:
If you do escape it, it doesn't render properly with Docusaurus. Pick your poison.

Reply:
I replaced curly brackets (`{`, `}`) with square brackets (`[`, `]`) that work on both GitHub and Docusaurus.

Reply:
Ok, let's keep it like this to have a working solution on both platforms, but keep in mind that it's wrong to use square brackets for sets. They are used for intervals, while more generic sets are defined using curly brackets.

Reply:
It doesn't work on Docusaurus when I build it. However, this works (without escaping `[`, `]`):
```
$$\left [ (x,y): x,y \in \mathbb{R}, y = a+bx \right ]$$
```
This is what is rendered when the square brackets are escaped:
![image](https://github.com/open-education-hub/ccas/assets/19818392/63364d05-ebb6-46cb-8243-7412f7b5e3fa)